daffodil-cli/src/main/scala/org/apache/daffodil/cli/Main.scala

```diff
@@ -226,14 +226,30 @@ class CLIConf(arguments: Array[String], stdout: PrintStream, stderr: PrintStream
   implicit def rootNSConverter = org.rogach.scallop.singleArgConverter[RefQName](qnameConvert _)
 
   implicit def fileResourceURIConverter = singleArgConverter[URI]((s: String) => {
```

Comment:
Why is there one of these resolvers in the CLI? So it can find the main schema on the classpath?

Add scaladoc motivating why this is needed.